sfmoe/react-challenge-project-jan-2023
# Userlogout

## Changes
_List Changes Introduced by this PR_
1. redux logout action added to nav.js
2.  browser navigates to welcome page on logout

## Purpose
Feature Request: Log Out  https://github.com/Shift3/react-challenge-project-jan-2023/issues/5

## Approach
user is logged out by clearing redux state and redirected to homepage


## Testing Steps
After log in user can click logout link and should be redirected to home page


_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #005
